pokemon/pokemon.py

```diff
@@ -0,0 +1,79 @@
+"""This module contains a two classes for creating pokemon."""
+
+
+class Pokemon(object):
+    """
+    This class Pokemon represents a pokemon character with several attributes.
+
+     :param name: Name of pokemon
+     :type name: str
+     :param nickname: Nickname of pokemon
+     :type nickname: str
+     :param moves: list of 4 moves
+     :type moves: list
+    """
+
+    def __init__(self, name, nickname):
+        """Initialise Pokemon class."""
+        self.name = name
+        self.nickname = nickname
+        self.moves = ["jump", "strike", "dash", "block"]
+
+    def speak(self):
+        """Pokemon say name."""
+        print("\n" + self.name.upper() + " " + self.name.upper() + "!")
+
+    def learn_move(self, new_move):
+        """Pokemon learn a new move."""
+        self.moves.append(new_move)
+        self.moves.pop(0)
```

Comment:
Today I learned that lists have `pop()` methods. I wasn't aware of that.

Just be careful how you use this. What is happening here is that you are appending a new move, and then popping the same new move right off the list!

If you want to push one of the old ones out, consider making the list of moves a `deque` rather (you need to `from collections import deque`) and then you can `popleft()`.

Reply:
Ooooh! Never mind, disregard this comment. I didn't notice the `0` initially, that specifically pops element number `0`/

My bad, the way you have it will work perfectly.